<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogCourse</class>
 <widget class="QDialog" name="dialogCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbCourseContent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>1251</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Contents</string>
   </property>
   <widget class="QTableView" name="tvCourse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1181</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbEditDataCourse">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1251</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit data</string>
   </property>
   <widget class="QLineEdit" name="txtCourseNumber">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCourseCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCourseDesc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course description:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCourseDesc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCourseCredit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCourseOutline">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course outline:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbCourseCredit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program code:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCourseAcaProgram">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QRadioButton" name="rdbtnAddCourse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Course</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnUpdateCourse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Course</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnDeleteCourse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Course</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushBtnCourse">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dynamic Button</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCourseOutline">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
